os/arch/arm/src/amebad/amebad_serial.c

```diff
@@ -808,7 +786,16 @@ int up_lowgetc(void)
  *  sent character
  *
  ****************************************************************************/
-int up_putc(int ch)
+
+#define MAX_BUFFER_SIZE 1000
```

Comment:
Don't have to consider size of uart buffer?

Reply:
This buffer is for holding the logs (lldbg) until we get to the end of the line and then the logs are output directly through up_lowputc. So I think we need not consider the size of uart buffer as this is not related to it.